### PR TITLE
Private link feedback

### PR DESCRIPTION
**Pull request by stevhliu**
Minor suggestions for clarity :)

### CODE REVIEW

File: docs/source/guides/private_link.mdx [hunk 1]
```diff
@@ -1,36 +1,35 @@
 # Create a Private Endpoint with AWS PrivateLink
 
-Security and secure inference are one of the key principles for Hugging Face Inference Endpoints. We are currently offering three different levels of security: [Public, Protected and Private](/docs/inference-endpoints/security).
+Security and secure inference are key features of Inference Endpoints. We currently offer three different levels of security: [Public, Protected and Private](/docs/inference-endpoints/security).
```
Comment (philschmid) [inline docs/source/guides/private_link.mdx:3]: isn't principle stronger than feature?

File: docs/source/guides/private_link.mdx [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
-Public and Protected Endpoints do not require any additional configuration, but to be able to create a Private Endpoint for a secure intra-region connection you need to provide the AWS Account ID of the Account, which also should have access to the Inference Endpoints.
+Public and Protected Endpoints do not require any additional configuration. But in order to create a Private Endpoint for a secure intra-region connection, you need to provide the AWS Account ID of the account which should also have access to Inference Endpoints.
 
 <img
   src="https://raw.githubusercontent.com/huggingface/hf-endpoints-documentation/main/assets/6_private_type.png"
   alt="select private link"
 />
 
-After you provide your AWS Account ID and click create Endpoint the Inference Service will create the VPC Endpoint. After some time you should see a VPC Service Name on the overview.
+After you provide your AWS Account ID and click **Create Endpoint** the Inference Service creates a VPC Endpoint which you'll see shortly in your overview.
```
Comment (philschmid) [inline docs/source/guides/private_link.mdx:12]: It is important to keep the correct words VPC service Name is not a VPC

File: docs/source/guides/private_link.mdx [hunk 3]
```diff
@@ -1,36 +1,35 @@
...
 
 <img
   src="https://raw.githubusercontent.com/huggingface/hf-endpoints-documentation/main/assets/6_service_name.png"
   alt="service link"
 />
 
-This VPC Service Name will be used to create the VPC Interface Endpoint in your (customer) Cloud account. Open console: https://console.aws.amazon.com/vpc/home?#Endpoints:`
+The VPC Service Name is used to create the VPC Interface Endpoint in your (customer) cloud account. Open your cloud account [console](https://console.aws.amazon.com/vpc/home?#Endpoints) to view the VPC Interface Endpoint.
```
Comment (philschmid) [inline docs/source/guides/private_link.mdx:19]: Opening the console will not view the Interface endpoint that's something you have to create

File: docs/source/guides/private_link.mdx [hunk 4]
```diff
@@ -1,36 +1,35 @@
...
-At this point, your Inference Endpoint has been created successfully, and when it is up and running you can go to the corresponding AWS Account and add the VPC Endpoint for the Endpoint.
+Once your Inference Endpoint is created successfully, go to the corresponding AWS account and add the VPC Endpoint as the endpoint.
 
 <img
   src="https://raw.githubusercontent.com/huggingface/hf-endpoints-documentation/main/assets/6_private_with_url.png"
   alt="endpoint url"
 />
 
-After your VPC Endpoint changed its status from “Pending” to “available” you should see a URL in the UI. This URL can now be used inside your VPC to access your endpoint in a secure, protected way ensuring that the traffic is only between the two endpoints and will never leave AWS.
+After the VPC Endpoint status changes from **Pending** to **available**, you should see a Endpoint URL in the overview. This URL can now be used inside your VPC to access your Endpoint in a secure and protected way, ensuring traffic is only occurring between the two endpoints and will never leave AWS.
```
Comment (philschmid) [inline docs/source/guides/private_link.mdx:35]:
```suggestion
After the VPC Endpoint status changes from **pending** to **available**, you should see a Endpoint URL in the overview. This URL can now be used inside your VPC to access your Endpoint in a secure and protected way, ensuring traffic is only occurring between the two endpoints and will never leave AWS.
```

**Review by philschmid — COMMENTED** [review]
Add some comments

File: docs/source/guides/private_link.mdx (hunk 1 above)
Comment (stevhliu) [reply docs/source/guides/private_link.mdx:3]: I think for me principle sounds more science-y or religious, and feature is more in line for describing a product. No strong feelings either way though :)

Comment (philschmid) [reply docs/source/guides/private_link.mdx:3]: lets keep principle

**Review by philschmid — APPROVED** [review]

**Pull request merged**